app/metrics.yaml

```diff
@@ -1545,6 +1545,19 @@ search.default_engine:
     expires: "2021-08-01"
 
 bookmarks_management:
+  open:
+    type: event
+    description: |
+      A user opened a bookmark.
```

Comment:
Can we add where the bookmark was opened from? I can imagine in the future we might want to extend top sites to allow bookmarks so it would be good to know where its source is from the description.

Reply:
We do track where the bookmark is opened from.  But for some reason we never tracked the normal use case where the user taps on the bookmark directly.  This is to fix it so we have all 5 cases covered.  The 5 cases have the following events:
open (User tapes on the bookmark directly)
open_in_new_tab (User taps on the 3 dot menu and opens the bookmark)
open_in_new_tabs (User selected multiple bookmark and then tap on the 3 dot menu to open the bookmarks)
open_in_private_tab (User taps on the 3dot menu to open the bookmark in a private tab)
open_in_private_tabs (User taps on the 3 dot menu to open multiple bookmarks in private tabs)

I added this sheet in the meta issue but forgot to include it here.  https://docs.google.com/spreadsheets/d/1EB6Mme0LR3GRE00YkX7Vm1VJOuTSnhwrSc4MOoySX5A/edit?usp=sharing.  This will help clear what event tracks what use case.  Thanks